lib/vagrant-seil/command.rb

```diff
@@ -0,0 +1,64 @@
+require "optparse"
+require "vagrant"
+
+module VagrantPlugins
+  module Seil
+    class Command < Vagrant.plugin(2, :command)
+      def self.synopsis
+        "save configuration of running SEIL into specified host file"
+      end
+
+      def execute
+        options = {}
+
+        opts = OptionParser.new do |o|
+          o.banner = "Usage: vagrant seil-save-to [options] [name]"
```

Comment:
Did you name the command "seil-save-to" (without "host")?  I like it better but do you think "seil-save-to-host" is better?